watertap/examples/flowsheets/mvc/mvc_single_stage.py

```diff
@@ -0,0 +1,745 @@
+    iscale.calculate_scaling_factors(m)
+
+    return m
+
+
+def add_Q_ext(m, time_point=None):
+    # Allows additional heat to be added to evaporator so that an initial feasible solution can be found as a starting
+    # guess for optimization in case physically infeasible simulation is proposed
+
+    if time_point is None:
+        time_point = m.fs.config.time
+    m.fs.Q_ext = Var(time_point, initialize=0, units=pyunits.J / pyunits.s)
+    m.fs.Q_ext[0].setlb(0)
+    m.fs.evaporator.eq_energy_balance.deactivate()
+    m.fs.evaporator.eq_energy_balance_with_additional_Q = Constraint(
+        expr=m.fs.evaporator.heat_transfer
+        + m.fs.Q_ext[0]
+        + m.fs.evaporator.properties_feed[0].enth_flow
+        == m.fs.evaporator.properties_brine[0].enth_flow
+        + m.fs.evaporator.properties_vapor[0].enth_flow_phase["Vap"]
+    )
+    iscale.set_scaling_factor(m.fs.Q_ext, 1e-6)
+
+
+def add_costing(m):
+    m.fs.costing = WaterTAPCosting()
+    m.fs.pump_feed.costing = UnitModelCostingBlock(flowsheet_costing_block=m.fs.costing)
+    m.fs.pump_distillate.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing
+    )
+    m.fs.pump_brine.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing
+    )
+    m.fs.hx_distillate.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing
+    )
+    m.fs.hx_brine.costing = UnitModelCostingBlock(flowsheet_costing_block=m.fs.costing)
+    m.fs.mixer_feed.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing
+    )
+    m.fs.evaporator.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing
+    )
+    m.fs.compressor.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing
+    )
+
+    m.fs.costing.cost_process()
+    m.fs.costing.add_annual_water_production(m.fs.distillate.properties[0].flow_vol)
+    m.fs.costing.add_LCOW(m.fs.distillate.properties[0].flow_vol)
+    m.fs.costing.add_specific_energy_consumption(m.fs.distillate.properties[0].flow_vol)
+
+
+def add_pressure_drop_to_hx(hx_blk, time_point):
+    # input: hx_blk - heat exchanger block
+    # output: deactivates control volume pressure balance and adds pressure drop to pressure balance equation
+    hx_blk.cold.pressure_balance.deactivate()
+    hx_blk.hot.pressure_balance.deactivate()
+    hx_blk.cold.deltaP = Var(time_point, initialize=7e4, units=pyunits.Pa)
+    hx_blk.hot.deltaP = Var(time_point, initialize=7e4, units=pyunits.Pa)
+    hx_blk.cold.pressure_drop = Constraint(
+        expr=hx_blk.cold.properties_in[0].pressure
+        == hx_blk.cold.properties_out[0].pressure + hx_blk.cold.deltaP[0]
+    )
+    hx_blk.hot.pressure_drop = Constraint(
+        expr=hx_blk.hot.properties_in[0].pressure
+        == hx_blk.hot.properties_out[0].pressure + hx_blk.hot.deltaP[0]
+    )
+
+
+def set_operating_conditions(m):
+    m.fs.costing.base_currency = pyo.units.USD_2020
```

Comment:
This line probably belongs in `add_costing`. Also, I think 2020 USD might be a little skewed due to the pandemic so considering an alternate year might be of interest for analysis purposes.